dartsim/src/Base.hh

```diff
@@ -291,6 +293,15 @@ class Base : public Implements3d<FeatureList<Feature>>
     return std::forward_as_tuple(id, entry);
   }
 
+  public: inline std::tuple<std::size_t, ModelInfo &> AddNestedModel(
+              const ModelInfo &_info, const std::size_t _parentID,
+              const std::size_t _worldID)
```

Comment:
Should we add a test for the `AddNestedModel` method in `Base_TEST.cc`?

Reply:
Added a test in https://github.com/ignitionrobotics/ign-physics/pull/228/commits/a652ef2b94b139322aa4db718d4e6606099634e1. Let me know if that's sufficient.

Reply:
Test look good, thanks!